src/v3/src/transformer/terminal/transformTerminalMessages.ts

```diff
@@ -143,6 +144,12 @@ export const transformTerminalMessages: TerminalKeyTransformer = (transaction, f
   } else if (containsMessageKeyPrefix('core.auth.factor.signedNonce.error', displayedMessages)) {
     // custom title for signed nonce errors
     displayedMessages[0].title = loc('user.fail.verifyIdentity', 'login');
+  } else if (containsMessageKeyPrefix('idx.error.code.access_denied.device_assurance.remediation', displayedMessages)) {
```

Comment:
Can you extract this `'idx.error.code.access_denied.device_assurance.remediation'` into a constant like the others?